apps/chronicles/screens/components/components/NavigationCellScreen.tsx

```diff
@@ -47,6 +47,43 @@ export const NavigationCellScreen = () => {
                 onPress={() => null}
             />
             <Spacer />
+            <View style={styles.readableContent}>
+                <Typography>...and even additional titles!</Typography>
+                <Typography>These can either be shown next to the title</Typography>
+            </View>
+            <Spacer />
+            <Cell.Navigation
+                title="This is a navigation cell with additional titles."
+                description="And this is still its description!"
+                additionalTitlesRight={[
+                    "Additional Title One",
+                    "Additional Title Two",
+                    "Additional Title Three",
+                ]}
+                iconName="navigation-outline"
+                onPress={() => null}
+            />
+            <Spacer />
+            <View style={styles.readableContent}>
+                <Typography>or under the description</Typography>
+            </View>
+            <Spacer />
+            <Cell.Navigation
+                title="This is a navigation cell with additional titles."
+                description="And this is still its description!"
+                additionalTitlesUnder={[
+                    "Additional Title One",
+                    "Additional Title Two",
+                    "Additional Title Three",
+                    "Additional Title Three",
+
+                    "Additional Title Three",
+                    "Additional Title Three",
```

Comment:
Nitpicking, but should there seems to be some duplicated title texts here. Maybe just increment the numbers instead of duplicating the titles so that we don't confuse anyone with the component not rendering the third properly?